render-to-string.ts

```diff
@@ -0,0 +1,60 @@
+import mdx, { Options } from '@mdx-js/mdx'
+import { MDXProvider, mdx as mdxReact } from '@mdx-js/react'
+import { transformAsync } from '@babel/core'
+import presetEnv from '@babel/preset-env'
+import presetReact from '@babel/preset-react'
+import { renderToString as reactRenderToString } from 'react-dom/server'
+import React from 'react'
+// @ts-ignore
```

Comment:
Just curious, why ignore this?

Reply:
Because the babel plugin module is written in JS and we don't have type definition for it, tsc complains.
I'm not sure where to find type definitions for babel plugins. Maybe someone can help.

Reply:
I provided `babel-plugin-mdx-browser.d.ts`: https://github.com/hashicorp/next-mdx-remote/pull/20/commits/1341796813f82d61216c422cabf6de804f74780f